Sources_v3/StreamChatUI/Utils/UIColor+Extensions.swift

```diff
@@ -37,5 +37,5 @@ extension UIColor {
     static let incomingMessageBubbleBackground = white
     static let incomingMessageBubbleBorder = UIColor.black.withAlphaComponent(0.08)
     static let chatBackground = UIColor(rgb: 0xfcfcfc)
-    static let messageTimestamp = UIColor.lightGray
```

Comment:
Please put this to `UIConfig`

Reply:
-> https://getstream.slack.com/archives/G019728A97W/p1607430740021300?thread_ts=1607430711.021000&cid=G019728A97W

Reply:
Will rebase the PR on main branch once color palette changes done by @ManWithBear are there